bibliopixel/main/all_pixel.py

```diff
@@ -56,7 +56,7 @@ def run(args, settings):
         print("Press Ctrl+C anytime to quit.")
 
         O("Scanning for devices...")
-        devs = sorted(Devices().find_serial_devices().items())
+        devs = sorted(Devices(args.hardware_id, args.baud).find_serial_devices().items())
```

Comment:
that line got a little long - consider perhaps extracting out `device = Devices(...)` as a separate line?